### DISCUSSION TITLE
How to Install CasaOS on existing setup?

### DISCUSSION THREAD

**Discussion by habibmy** [IceWhaleTech/CasaOS#1387] [Q&A]
I have a homeserver and it is running debian. I also use docker on it with 40+ containers. I also want to install CasaOS alongside my existing docker containers.

**Comment by thedroidkid** [comment]
You can't run CasaOS in a docker however, you can install it on your Debian system. Just note it will install to port 80

**Reply by habibmy** [reply]
will it affect my existing containers?
